server/config/express.js

```diff
@@ -14,6 +13,9 @@ module.exports.init = function() {
   //initialize app
   var app = express();
 
+  // Readable JSON output
+  app.set('json spaces', 4);
```

Comment:
Better JSON output. Although not necessarily need, better for general debugging JSON